{{ cookiecutter.project_slug }}/pyproject.toml

```diff
@@ -6,20 +6,20 @@ authors = ["{{ cookiecutter.project_author_name }}"]
 license = "{{ cookiecutter.project_license }}"
 
 [tool.poetry.dependencies]
-python = "^3.8"
-click = "^8.1.3"
-loguru = "^0.6.0"
-PyYAML = "^6.0"
+python = ">3.8,<4"
```

Comment:
Warum nicht "^3.8"?

https://python-poetry.org/docs/dependency-specification/

Reply:
Jeep, that's a mistake. Should be >=3.8.

Reply:
```suggestion
python = "^3.8"
```